### PR TITLE
Clarify ipv6 option for minion and inteface for master, closes #39118

### LINKED ISSUE #39118: Minion ipv6 option is not documented
### Description of Issue/Question

Salt master docs: https://docs.saltstack.com/en/latest/ref/configuration/master.html

image

Salt minion docs:

image

Salt minion docs say that `master` is `ipv4` address and don't mention `ipv6`.

When running `salt-call --master=ipv6.google.com test.ping` salt fails:

```
[ERROR   ] DNS lookup of 'False' failed.
[ERROR   ] Master hostname: 'ipv6.google.com' not found. Retrying in 30 seconds
```

This is because even though `salt-call` accepts `--master` it uses `ipv6: false` (the default).

### Versions Report

```
Salt Version:
           Salt: 2016.3.4

Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.2
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: Not Installed
      pycparser: 2.10
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5

System Versions:
           dist: debian 8.6
        machine: x86_64
        release: 4.4.44
         system: Linux
        version: debian 8.6
```

### PR DESCRIPTION
**Pull request by bobrik**

### CODE REVIEW

File: doc/ref/configuration/minion.rst [hunk 1]
```diff
@@ -77,6 +78,17 @@ The option can can also be set to a list of masters, enabling
 
 .. conf_minion:: master_type
 
+``ipv6``
+--------
+
+Default: ``False``
+
+Whether the master should be connected over IPv6. If this is set to True
```
Comment (rallytime) [inline doc/ref/configuration/minion.rst]: Was there more you were going to include as part of this sentence?

**Review by rallytime — COMMENTED** [review]
Thanks for grabbing this @bobrik! I had one quick question, otherwise this looks good.

**Comment by rallytime** [comment]
Fixes #39118

File: doc/ref/configuration/minion.rst (hunk 1 above)
Comment (bobrik) [inline doc/ref/configuration/minion.rst]: Oops, should be good now.

**Comment by rallytime** [comment]
Awesome! thanks!

**Review by rallytime — APPROVED** [review]

**Pull request merged**